test/fixtures/transactions.js

```diff
@@ -51,6 +51,10 @@ const Transaction = stampit({
 		this.type = type || 0;
 
 		switch (this.type) {
+			case 2:
+				this.asset.delegate.username = delegateName || '';
```

Comment:
@Tschakki Because any delegate can't be a without a username. So instead of assigning empty string, generate some random username if not provided.